src/commands/build.ts

```diff
@@ -24,40 +24,10 @@ import {PolymerProject} from 'polymer-build';
 import {applyBuildPreset, ProjectBuildOptions, ProjectConfig} from 'polymer-project-config';
 
 import * as buildLibTypeOnly from '../build/build';
-
 import {Command, CommandOptions} from './command';
 
-
 const logger = logging.getLogger('cli.command.build');
 
-/**
- * Converts command-line build arguments to the `ProjectBuildOptions` format
- * that our build understands, applying the preset if one was given.
- */
-function commandOptionsToBuildOptions(options: CommandOptions):
-    ProjectBuildOptions {
-  return applyBuildPreset({
```

Comment:
By defining all options in the argument, `applyBuildPreset()` interpreted them all as overrides on the preset.